crowdin.yml

```diff
@@ -13,3 +13,9 @@ files:
   - source: /main/project/playstore/description/_source/*.txt
     translation: /main/project/playstore/description/translated/%language%/%original_file_name%
     content_segmentation: 1
+  - source: /main/res/raw/changelog_master.xml
+    translation: /main/res/raw-%two_letters_code%/%original_file_name%
+    content_segmentation: 1
+  - source: /main/res/raw/changelog_release.xml
+    translation: /main/res/raw-%two_letters_code%/%original_file_name%
```

Comment:
They are now *.md instead of *.xml, isn´t it?

Reply:
Ah, good catch, thanks. Will adapt the PR.